src/domains/ValuesDomain.js

```diff
@@ -326,7 +334,12 @@ export default class ValuesDomain {
 
   // Note that calling this can result in user code running, which can side-effect the heap.
   // If that is not the desired behavior, mark the realm as read-only for the duration of the call.
-  static computeLogical(realm: Realm, op: BabelNodeLogicalOperator, lval: ConcreteValue, rval: ConcreteValue): Value {
+  static computeLogical(
+    realm: Realm,
+    op: BabelNodeLogicalOperator,
+    lval: ConcreteValue,
+    rval: ConcreteValue
+  ): ConcreteValue {
```

Comment:
A logical operator with two concrete inputs is never abstract as its return type.